functional/streams.py

```diff
@@ -174,8 +175,28 @@ def json(json_file):
         return seq(six.viewitems(json_input))
 
 
+def sqlite3(conn, sql):
+    """
+    Additional entry point to Sequence which query data from sqlite db file.
+
+    :param conn: path or sqlite connection, cursor
+    :param sql: SQL query string
+    :return: Sequence wrapping SQL cursor
+    """
+
+    if isinstance(conn, (sqlite3api.Connection, sqlite3api.Cursor)):
+        input_conn = conn
+    elif isinstance(conn, str):
+        input_conn = sqlite3api.connect(conn)
```

Comment:
Connection should get cleaned up using `with`. Could change this line to return `seq` here to avoid introducing more variables. Something like

```python
with sqlite3api.connect(conn) as input_conn:
    return seq(input_conn.execute(sql))
```